src/rkopenmdao/error_controllerx.py

```diff
@@ -0,0 +1,139 @@
+from rkopenmdao.error_controller import ErrorController
+
+
+__all__ = [
+    "Integral",
+    "H0_110",
+    "H_211",
+    "H0_211",
+    "PC",
+    "H0_220",
+    "PID",
+    "H_312",
+    "H0_312",
+    "H_312_general",
+    "PPID",
+    "H_321",
+    "H0_321",
+    "H_321_general",
+    "H0_330",
+    "H_330_general",
+]
```

Comment:
The above isn't necessary with the way the file is currently written. This __all__ = [...] just restrits what is imported when you do

`from error_controllerx.py import *`.

Since your __all__ contains everything that is defined in this file, it is unnecessary in the way it is now.

Reply:
all will be removed